kubecontrolplane/v1/types.go

```diff
@@ -62,6 +62,12 @@ type KubeAPIServerConfig struct {
 
 	// TODO this needs to be removed.
 	APIServerArguments map[string]Arguments `json:"apiServerArguments"`
+
+	// MinimumKubeletVersion is the lowest version of a kubelet that can meaningfully join the cluster.
+	// Specifically, the apiserver will deny (most) authorization requests of kubelets that are older
```

Comment:
What's the expectation when this field and the one in v1Node contradict each other and how can an API consumer understand it?